app/controllers/api/v1/products_controller.rb

```diff
@@ -57,4 +56,8 @@ def product_params
     params.permit([:name, :price, :quantity])
```

Comment:
I'd use require of  the model, then permit of the whitelisted parameters (to be strict about the format of the payload). Do you have a strong reason not to?

Reply:
Yes, you are correct.